### DISCUSSION TITLE
产品方便性的小建议

### DISCUSSION THREAD

**Discussion by Metugit** [OtterMind/Chat2DB#759] [General]
非常喜欢这个产品，但是在使用上有些小不便。我想提出一些小建议来供参考。
1、数据库连接信息，能否加上分组功能，连接信息太多的时候有些混乱不好找。
2、目前sql控制台是跟着数据库的，当我切换数据库的时候控制台也会随着消失，虽说切换回去的时候能有保留或者通过保存来恢复。但是还是希望将控制台窗口独立于库。

**Comment by taozuhong** [comment]
袋鼠数据库工具 支持了你的建议，但还没有支持拖动编辑功能，估计后续会加上；

**Comment by openai0229** [comment]
现在都支持了哦

> 非常喜欢这个产品，但是在使用上有些小不便。我想提出一些小建议来供参考。 1、数据库连接信息，能否加上分组功能，连接信息太多的时候有些混乱不好找。 2、目前sql控制台是跟着数据库的，当我切换数据库的时候控制台也会随着消失，虽说切换回去的时候能有保留或者通过保存来恢复。但是还是希望将控制台窗口独立于库。
